src/Components/CuriosityHero/CuriosityHero.tsx

```diff
@@ -1,4 +1,5 @@
 ﻿﻿import React from "react";
+import 'CuriosityHero.css';
```

Comment:
could you change `CuriosityHero` to be a `scss` file?

Reply:
changed